gax-java/gax/src/test/java/com/google/api/gax/rpc/ClientContextTest.java

```diff
@@ -30,10 +30,7 @@
 package com.google.api.gax.rpc;
 
 import static com.google.common.truth.Truth.assertThat;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
+import static org.junit.Assert.*;
```

Comment:
You may need to adjust the intellij settings to not import `*`, see this [guide](go/java-style#s3.3.1-wildcard-imports) for no wildcard imports.

Reply:
Thanks, removing